### ISSUE TITLE
Let me approve Russian translations for OSARA

### ISSUE THREAD

**Issue by Kostenkov-2021** [jcsteh/osara#1129]
Good day. Please provide to me the ability of approveing a new Russian strings off OSARA on crowdin platform. My crowdin Login is Dan_kost_2023.

**Comment by jcsteh** [comment]
You should already have full access to manage the Russian translation, as requested and granted in #875. Can you explain what's not working?

**Comment by Kostenkov-2021** [comment]
When loading completed translation, I have not checkbox "approve added translations"as it happens for example in NVDA project..

**Comment by jcsteh** [comment]
I don't think you should need to approve translations for OSARA. They should just be applied as long as you save the strings. Is this not the case for you? Can you give me specific strings where you're seeing problems as examples so I can take a look?

**Comment by jcsteh** [comment]
Closing, as I believe everything to be working as expected and I haven't had a response in over 6 months. Please reopen if you're still having problems updating your translation. Thanks.
